dex/testing/doge/harness.sh

```diff
@@ -273,6 +277,8 @@ tmux send-keys -t $SESSION:0 C-c
 tmux send-keys -t $SESSION:1 C-c
 tmux send-keys -t $SESSION:2 C-c
 tmux send-keys -t $SESSION:3 C-c
+tmux send-keys -t $SESSION:4 C-c
```

Comment:
This tries to kill the `quit` script itself.  A few lines down it will remove the ctl session using `tmux kill-session`.

Reply:
Removed this ctrl-c.